src/services/WolfDen.Domain/Entity/Status.cs

```diff
@@ -0,0 +1,29 @@
+﻿
+
+namespace WolfDen.Domain.Entity
+{
+    public class Status
+    {
+        public int Id { get;private set; }
+        public int EmployeeId { get;private set; }

```

Comment:
Why employee Id in this column

Reply:
using employeeId and date ,status of each day is taken to display it in the calender.

Reply:
For this you don't need a table, you have daily attendance table ?

Reply:
But only if he is present or late,there will be an entry in the table .so to show status like holiday ,wfh ,...status is updated in the status table

Reply:
You can always join those tables, don't need to prepare this table

Reply:
if a punch is there,there will be an entry in the daily attendance .so that status of that will be stored here.But if not there is no other table which stores this status other than this status table.So there is no other table to join.

Reply:
When will this entry is populated , you can always check leave and holiday related tables to build the query

Reply:
ok.We thought this can be scheduled so that the entry is populated the next day like the email is going to sent .Because in every task like weekly,daily monthly attendance summaries and reports,status is a part and  the logic will be repeated.